lib/builders/observability-builder.ts

```diff
@@ -1,24 +1,26 @@
-import * as blueprints from '../../lib';
+import * as blueprintbuilder from '../../lib/stacks';
```

Comment:
let's replace with `import { BlueprintBuilder } from ...` and just use it as BlueprintBuilder

Reply:
fixed